tutorials/colors/colorbar_only.py

```diff
@@ -85,7 +85,7 @@
 #
 # Here we illustrate the use of custom length colorbar extensions, used on a
 # colorbar with discrete intervals. To make the length of each extension same
-# as the length of the interior colors, use extendfrac='auto'.
+# as the length of the interior colors, use ``extendfrac='auto'``.
```

Comment:
actually do you think you could make this triple backticks?

...JK, this PR looks good to me :)